packages/web-react/src/hooks/useDeprecatedMessage.ts

```diff
@@ -54,26 +59,29 @@ export const useDeprecationMessage = ({
 }: UseDeprecationMessageProps) => {
   let message: string;
   let hasProps: boolean;
+  const messageBase = `Deprecation warning (${componentName}):`;
 
   useEffect(() => {
     const isExecutable = trigger && componentName && process.env.NODE_ENV === 'development';
 
     switch (method) {
       case 'property':
         if (propertyProps?.delete) {
-          message = `Deprecation warning (${componentName}): "${propertyProps?.deprecatedName}" property will be deleted in next major version..️️`;
+          message = `${messageBase} "${propertyProps?.deprecatedName}" property will be deleted in next major version..️️`;
+        } else if (propertyProps?.deprecatedValue && propertyProps?.newValue && propertyProps?.propertyName) {
+          message = `${messageBase} This "${propertyProps?.deprecatedValue}" value for "${propertyProps?.propertyName}" property will be renamed to "${propertyProps?.newValue}" in the next major version. Use this instead.️️`;
```

Comment:
See my comment in twig about the text. I think it could be improved.

Reply:
done